src/components/UserDetailSections/PatronBlock/PatronBlock.js

```diff
@@ -158,9 +160,23 @@ class PatronBlock extends React.Component {
       },
     } = this.props;
 
+    const pointerWrapper = (content, isManual) => (
+      isManual
+        ? <div className={css.pointerWrapper}>content</div>
```

Comment:
It should be `{content}` here with curly braces or we will receive `content` as plain text

Reply:
Fixed